domain.yml

```diff
@@ -15,63 +37,95 @@ entities:
-  utter_ask_exercise:
-  - text: What kind of exercise did you do 💪 ?
-  utter_ask_sleep:
-  - text: "How much sleep did you get 💤  last night?"
-  utter_ask_diet:
-  - text: "Did you stick to a healthy diet 🥦  yesterday?"
-  utter_ask_stress:
-  - text: "What is your stress level right now 🧘 ?"
-    buttons:
-      - title: "low"
-        payload: '/inform{"stress": "low"}'
-      - title: "medium"
-        payload: '/inform{"stress": "medium"}'
-      - title: "high"
-        payload: '/inform{"stress": "high"}'
-  utter_ask_goal:
-  - text: "Setting goals - even small ones - is a great way to focus your day. What do you want to accomplish today 🥇 ?"
-  utter_slots_values:
-  - text: "You've submitted the following answers:\n
+
+  utter_ask_health_form_sleep:
+  - text: How much sleep did you get 💤 last night?
+
+  utter_ask_health_form_exercise :
+  - text: What kind of exercise did you do?
+
+  utter_ask_health_form_diet:
+  - text: Did you stick to a healthy diet 🥦 yesterday?
+
+  utter_ask_health_form_stress:
+  - text: Is your stress level low, medium or high 🧘🏼?
+
+  utter_ask_health_form_goal:
+  - text: Setting goals - even small ones - is a great way to focus your day. What do you want to accomplish today 🥇 ?
+
+  utter_slots_value:
+  - text: "Here's your daily wellness log:\n
             - Exercised?: {confirm_exercise}\n
             - Type of exercise: {exercise}\n
             - Sleep: {sleep}\n
             - Stuck to a healthy diet?: {diet}\n
             - Stress level: {stress}\n
             - Goal: {goal}"
+
   utter_no_worries:
-  - text: "No problem :)"
+  - text: No Problem :)
+
   utter_ask_continue:
-  - text: "Sorry, I don't quite understand. Do you want to continue?"
+  - text: Sorry, I don't quire understand. Do you want to continue?
 
-slots:
-  confirm_exercise:
-    type: unfeaturized
-  exercise:
-    type: unfeaturized
-  sleep:
-    type: unfeaturized
-  diet:
-    type: unfeaturized
-  stress:
-    type: unfeaturized
-  goal:
-    type: unfeaturized
+actions:
+- validate_health_form
+
+forms:
+  health_form:
+    confirm_exercise:
+    - type: from_intent
+      intent: inform
+      value: true
+    - type: from_intent
+      intent: affirm
+      value: true
+    - type: from_intent
+      intent: deny
+      value: false
+    exercise:
+    - type: from_entity
+      entity: exercise
+    sleep:
+    - type: from_entity
+      entity: sleep
+    - type: from_intent
+      intent: deny
+      value: "None"
+    diet:
+    - type: from_text
+      intent: inform
+    - type: from_text
+      intent: affirm
+    - type: from_text
+      intent: deny
+    stress:
+    - type: from_entity
+      entity: stress
+    goal:
+    - type: from_text
+      intent: inform
 
 session_config:
   session_expiration_time: 60
-  carry_over_slots_to_new_session: false
-
-forms:
-  - health_form
\ No newline at end of file
+  carry_over_slots_to_new_session: true
```

Comment:
Curious about why you'd want to have slots carry over to new sessions? The use case is to have the user fill this form out once per day, so it seems like you'd want it to re-set after each session rather than each form-fill. I see you've also added an action_restart to each story - wondering what the reasoning is there?

Reply:
`action_restart` is for resetting the form, in case user wants to retry filling the form then we would want bot to recollect fresh responses instead of using information from prev interaction.

I see your point for `carry_over_slots_to_new_session: true` I am making it false :)